packages/design/components/Input/index.tsx

```diff
@@ -16,20 +12,49 @@ export interface InputProps {
   prefix?: React.ReactNode;
   /** 后缀 */
   suffix?: React.ReactNode;
-}
+  /** 对齐方式 */
+  alight?: 'right' | 'left';
+};
+
+type IInput = React.ForwardRefExoticComponent<InputProps> & {
+  Group: React.FC<JSX.IntrinsicElements['div']>;
+};
 
-/* eslint-disable react/prop-types */
 // https://github.com/jsx-eslint/eslint-plugin-react/issues/3140
-const Input = forwardRef<HTMLInputElement, InputProps>(
-  ({ type = 'text', wrapperStyle, wrapperClass, prefix, suffix, placeholder, ...rest }, ref) => {
-    return (
-      <div className={classnames('bgm-input__wrapper', wrapperClass)} style={wrapperStyle}>
-        {prefix}
-        <input type={type} className='bgm-input' placeholder={placeholder} ref={ref} {...rest} />
-        {suffix}
-      </div>
-    );
-  },
-);
+const Input = forwardRef<HTMLInputElement, InputProps>(function InputWithRef(
+  { type = 'text', wrapperStyle, wrapperClass, prefix, suffix, alight, disabled, ...rest },
+  ref,
+) {
+  return (
+    <div
+      className={classnames(
+        'bgm-input__wrapper',
+        disabled && 'bgm-input__wrapper--disabled',
```

Comment:
```suggestion
        {
          'bgm-input__wrapper--disabled': disabled
        },
```

带判断的一般推荐这么写，下面那个 align-right（不是 alight 也一样